testsuites/syncgateway/functional/tests/test_cbs_collections.py

```diff
@@ -506,7 +504,169 @@ def test_import_filters(scopes_collections_tests_fixture):
+    assert third_collection_stats["sync_function_reject_count"] == 1, f"{third_collection} sync_function_reject_count should be 1, got {third_collection_stats['sync_function_reject_count']}"
+
+    assert second_collection_stats["sync_function_reject_access_count"] == 1, f"{second_collection} sync_function_reject_access_count should be 1, got {second_collection_stats['sync_function_reject_access_count']}"
+    assert third_collection_stats["sync_function_reject_access_count"] == 1, f"{third_collection} sync_function_reject_access_count should be 1, got {third_collection_stats['sync_function_reject_access_count']}"
+
+    assert second_collection_stats["sync_function_exception_count"] == 0, f"{second_collection} sync_function_exception_count should be 0, got {second_collection_stats['sync_function_exception_count']}"
+    assert third_collection_stats["sync_function_exception_count"] == 1, f"{third_collection} sync_function_exception_count should be 0, got {third_collection_stats['sync_function_exception_count']}"
+
+    assert second_collection_stats["import_count"] == 2, f"{second_collection} import_count should be 2, got {second_collection_stats['import_count']}"
+    assert third_collection_stats["import_count"] == 1, f"{third_collection} import_count should be 1, got {third_collection_stats['import_count']}"
+
+    assert second_collection_stats["num_doc_reads"] == 2, f"{second_collection} num_doc_reads should be 2, got {second_collection_stats['num_doc_reads']}"
+    assert third_collection_stats["num_doc_reads"] == 1, f"{third_collection} num_doc_reads should be 1, got {third_collection_stats['num_doc_reads']}"
+
+    assert second_collection_stats["doc_reads_bytes"] > 0, f"{second_collection} doc_reads_bytes should be greater than 0, got {second_collection_stats['doc_reads_bytes']}"
+    assert third_collection_stats["doc_reads_bytes"] > 0, f"{third_collection} doc_reads_bytes should be greater than 0, got {third_collection_stats['doc_reads_bytes']}"
+
+    assert second_collection_stats["num_doc_writes"] == 5, f"{second_collection} num_doc_writes should be 5, got {second_collection_stats['num_doc_writes']}"
+    assert third_collection_stats["num_doc_writes"] == 4, f"{third_collection} num_doc_writes should be 4, got {third_collection_stats['num_doc_writes']}"
+
+    assert second_collection_stats["doc_writes_bytes"] > 0, f"{second_collection} doc_writes_bytes should be greater than 0, got {second_collection_stats['doc_writes_bytes']}"
+    assert third_collection_stats["doc_writes_bytes"] > 0, f"{third_collection} doc_writes_bytes should be greater than 0, got {third_collection_stats['doc_writes_bytes']}"
+
+
 def rename_a_single_scope_or_collection(db, scope, new_name):
     data = {"bucket": bucket, "scopes": {scope: {"collections": {new_name: {}}}}, "num_index_replicas": 0}
     admin_client.post_db_config(db, data)
     admin_client.wait_for_db_online(db, 60)
+
+
+def try_get_collection_stats(db, scope, collection, stats):
```

Comment:
You can consider removing the try from the name, as it gives information about the implementation, rather than a name the explains the general functionality.

Reply:
Corrected